db/migrate/20240621125508_match_route_eleven_translations.rb

```diff
@@ -0,0 +1,28 @@
+class MatchRouteElevenTranslations < ActiveRecord::Migration[7.0]
+  def up
+    translations_eleven.each do |k, v|
+      Translation.create!(
+        key: k,
+        text: v,
+      )
+    end
+  end
+
+  def down
+    Translation.where(key: translations_eleven.keys).destroy_all
+  end
+
+  private
+
+  def translations_eleven = {
+    'Match Route Eleven' => 'Match Route Eleven',
+    'SSP Eleven' => 'SSP',
+    'Shelter Agency Eleven' => 'Shelter Agency',
+    'HSA Eleven' => 'HSA',
+    'Housing Search Provider Eleven' => 'Housing Search Provider',
+    'Housing Subsidy Administrator Eleven' => 'Housing Subsidy Administrator',
+    'Stabilization Service Provider Eleven' => 'Stabilization Services Provider',
+    'Stabilization Services Provider Eleven' => 'Stabilization Services Provider',
+    'CoC Eleven' => 'DND',
+  }
+end
```

Comment:
Hmm, I could be wrong, and maybe we want some default translations in there, but I _think_ you can leave this off and the app will add them via a cron job and/or if you visit something that needs it.  (It would just add the untranslated value though, so maybe something like this is useful.)

Reply:
From my understanding the app will automatically add these transalations. I'll remove this file and we can add it back in if needed. I saw a previous match route setting the default translations via script so included it here based on that knowledge.